logdetective/server/models.py

```diff
@@ -118,6 +119,37 @@ def __init__(self, data: Optional[dict] = None):
         self.max_artifact_size = int(data.get("max_artifact_size")) * 1024 * 1024
 
 
+LogLevel = Literal[
+    "CRITICAL",
+    "FATAL",
+    "ERROR",
+    "WARN",
+    "WARNING",
+    "INFO",
+    "DEBUG",
+    "NOTSET",
```

Comment:
Is there a reason to redefine these rather than just use `logging.CRITICAL`, `logging.DEBUG`, etc?

When setting loglevel (either with `basicConfig()` or `logger.setlevel()`) we can just pass `levelstring.upper()` and it will be interpreted properly. What does this redefinition gain us?

Reply:
The reason was to have a specific type here:

```python
class LogConfig(BaseModel):
    ...
    level: LogLevel = "INFO"
```

But if you want, we can drop the list and use:
```python
class LogConfig(BaseModel):
    ...
    level: str | int = "INFO"
```